thrift/user-feedback.md

```diff
@@ -0,0 +1,20 @@
+## Issues
+
+If you have any problems with or questions about this image, please contact me
+through a [GitHub issue](https://github.com/ahawkins/docker-thrift/issues).
+
+You can also reach many of the official image maintainers via the
+`#docker-library` IRC channel on [Freenode](https://freenode.net).
+
+## Contributing
+
+You are invited to contribute new features, fixes, or updates, large
+or small; I am always thrilled to receive pull requests, and do my
+best to process them as fast as I can.
+
+Before you start to code, I recommend discussing your plans through a
+[GitHub issue](https://github.com/ahawkins/docker-thrift/issues),
+especially for more ambitious contributions. This gives other
+contributors a chance to point you in the right direction, give you
+feedback on your design, and help you find out if someone else is
+working on the same thing.
```

Comment:
This file seems like it's pretty close to the standard template -- is that right?